native/fhir-to-bal-template/src/main/java/org/wso2/healthcare/fhir/codegen/ballerina/project/tool/config/BallerinaProjectToolConfig.java

```diff
@@ -207,6 +194,6 @@ public String getServicePackage() {
         return servicePackage;
     }
     public String getResourcePackage() {
```

Comment:
```suggestion

    public String getDependentPackage() {
```

Change the usages too

Reply:
done